Update (#60)
Drakkar-Software/OctoBot-Project-Website: _i18n/en/_posts/2022-12-21-cloud_octobot.md

---
layout: post
short_name: cloud_octobot
title: The OctoBot Cloud is live
subtitle: Easily deploy your OctoBot on the OctoBot Cloud.
date: 2022-12-21
image: posts/cloud-bot-banner.jpg
link: https://blog.octobot.online/deploy-octobots-in-astrolab
categories: latest_new
external: true
---

> Simply deploy your OctoBot on our cloud platform and enjoy an Octobot that is always online and accessible from anywhere.

Reviewer: ```suggestion
Simply deploy your OctoBot on our cloud platform and enjoy it from anywhere at everytime.
```
